rules/o365/defense_evasion_o365_exchange_malware_filter_rule_mod.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/11/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a malware filter rule has been deleted or disabled in Office 365. An adversary or malicious insider may
+want to modify a malware filter rule to evade detection.
```

Comment:
Is "insider threat" the well-known term?

Reply:
updated to insider threat 🏈